### DISCUSSION TITLE
GLB 3D model jumping up suddenly when opened in AR mode.

### DISCUSSION THREAD

**Discussion by rksambari** [google/model-viewer#3809] [Q&A]
Hi,

We have used google model viewer as below with a GLB 3D model but when we opened it in AR mode, the 3D model is jumping up, and it is happening only for the GLB file and it's working fine for the other format files like gltf and usdz. In the below code, we have used angular js directives(ng-src).

`<model-viewer id="modelId" bounds="tight" enable-pan ng-src="{{glb | trusted}}" ng-ios-src="{{glb | trusted}}" ar ar-modes="webxr" camera-controls environment-image="neutral" shadow-intensity="1"></model-viewer>`

Thanks.

**Comment by elalish** [comment]
I don't think this is a difference between GLB and glTF, since they produce the same data. And we don't process USDZ; can you specify which of our three AR modes you're in? If not, we can figure it out if you tell us the device and show a screen recording. Wild guess is you're seeing [this](https://github.com/google/model-viewer/discussions/1885#discussioncomment-3646769) behavior.

**Reply by rksambari** [reply]
Hi,
Please check the video record, the object moves upside suddenly without my action on it.

I was testing in **Android mobile**, AR mode is **webxr**

**Reply by elalish** [reply]
Thanks for the recording! Indeed, this is the expected behavior when the floor is found by ARCore (it wasn't possible until you moved your phone side-to-side, hence why a prompt is helpful - see https://modelviewer.dev/examples/augmentedreality/index.html#webXR). It looks like the problem is that your wine bottle is huge, likely authored in the wrong units, so it is very far away from you. It appears to be several meters tall. glTF specifies all files are in meters, so be careful on export. You might try a quick fix using [`scale`](https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-scale).

**Reply by rksambari** [reply]
Hi elalish,

Thanks for the update, I will try and get back.
